backend/src/test/java/com/woowacourse/momo/controller/meeting/MeetingControllerTest.java

```diff
@@ -31,20 +35,52 @@ class MeetingControllerTest {
     @LocalServerPort
     private int port;
 
+    @Autowired
+    private MeetingRepository meetingRepository;
+
     @Autowired
     private AttendeeRepository attendeeRepository;
 
     @Autowired
-    private MeetingRepository meetingRepository;
+    private AvailableDateRepository availableDateRepository;
 
     private Meeting meeting;
+    private List<String> dates;
 
     @BeforeEach
     void setUp() {
         RestAssured.port = port;
 
-        meeting = meetingRepository.save(MeetingFixture.COFFEE.create());
-        attendeeRepository.save(AttendeeFixture.HOST_JAZZ.create(meeting));
+        meeting = meetingRepository.save(MeetingFixture.DINNER.create());
+        List<AvailableDate> availableDates = List.of(
+                availableDateRepository.save(new AvailableDate(LocalDate.now(), meeting)),
+                availableDateRepository.save(new AvailableDate(LocalDate.now().plusDays(1), meeting))
+        );
+        dates = availableDates.stream()
+                .map(AvailableDate::getDate)
+                .map(LocalDate::toString)
+                .toList();
+    }
+
+    @DisplayName("UUID로 약속 정보를 조회하는데 성공하면 200 상태 코드를 응답한다.")
+    @Test
+    void find() {
+        Response response = RestAssured.given().log().all()
+                .contentType(ContentType.JSON)
+                .when().get("/api/v1/meeting/{uuid}", meeting.getUuid());
+
+        String meetingName = response.jsonPath().getString("data.meetingName");
+        String firstTime = response.jsonPath().getString("data.firstTime");
+        String lastTime = response.jsonPath().getString("data.lastTime");
+        List<String> availableDatesList = response.jsonPath().getList("data.availableDates", String.class);
+
+        assertSoftly(softAssertions -> {
+            softAssertions.assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+            softAssertions.assertThat(meetingName).isEqualTo(meeting.getName());
+            softAssertions.assertThat(firstTime).isEqualTo(meeting.startTimeslotTime().toString());
+            softAssertions.assertThat(lastTime).isEqualTo(meeting.endTimeslotTime().toString());
+            softAssertions.assertThat(availableDatesList).containsExactlyElementsOf(dates);
+        });
     }
```

Comment:
```suggestion
    @DisplayName("UUID로 약속 정보를 조회하는데 성공하면 200 상태 코드를 응답한다.")
    @Test
    void find() {
        RestAssured.given().log().all()
                .contentType(ContentType.JSON)
                .when().get("/api/v1/meeting/{uuid}", meeting.getUuid())
                .then().log().all()
                .statusCode(HttpStatus.OK.value())
                .body(
                        "data.meetingName", equalTo(meeting.getName()),
                        "data.firstTime", equalTo(meeting.startTimeslotTime().toString()),
                        "data.lastTime", equalTo(meeting.endTimeslotTime().toString()),
                        "data.availableDates", equalTo(dates)

                );
    }
```

이렇게 RestAssured만을 사용해서 응답의 데이터를 검증할 수 있어요!

참고 : [RestAssured-verify-body](https://github.com/rest-assured/rest-assured/wiki/Usage#example-1---json)

Reply:
저는 API를 호출과 검증 컨텍스트를 나눠 인수 테스트를 작성하는 걸 선호합니다.

Reply:
저는 API 호출과 검증 컨텍스트를 분리해서 인수 테스트를 작성하는 걸 선호합니다.

Reply:
좋습니다~ 팀원들과 다시 이야기 해보는 것이 좋을 것 같네요..!